cmd/root.go

```diff
@@ -107,4 +108,5 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", cfgFile, "config file")
 	rootCmd.PersistentFlags().StringVar(&logLevel, "loglevel", "info", "log level")
+	rootCmd.PersistentFlags().BoolVarP(&slack, "slack", "s", false, "Slack formatting")
```

Comment:
this should be in the List command since it's really only a flag on that command, not all of them for it being applied on the toot.